pysparkling/fileio/fs/s3.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import absolute_import
 
-import fnmatch
+from fnmatch import fnmatch
 import logging
-from io import BytesIO, StringIO
```

Comment:
The actual diff is to not to remove this line, but to replace it with `from io import BytesIO`, because `StringIO` is not used in this module.